tests/test_batch.py

```diff
@@ -1,71 +1,91 @@
 from unittest.mock import Mock, MagicMock, patch
 
+import json
+import os
 import pytest
+import requests_mock
 
 from sdk import SDK
 from sdk.batch import Batch
 
 
+TEST_API_FIXTURES_PATH = "tests/fixtures/api"
+JSON_FILE = "_.{}.json"
+
+
 class TestBatch:
     @pytest.fixture(autouse=True)
-    @patch("sdk.Client")
-    def init_sdk(self, mock_client):
-        self.sdk = SDK()
-        self.pulser_sequence = "pulser_test_sequence"
+    @requests_mock.Mocker(kw="mock")
+    def init_sdk(self, **kwargs):
```

Comment:
nit: I would deconstruct the "mock" kwarg from the dictionnary

`def init_sdk(self, mock=None, **kwargs)`